features/system-report/src/test/java/org/opennms/systemreport/system/ReportPluginITCase.java

```diff
@@ -66,15 +66,8 @@
 
     @Before
     public void setUp() {
-        System.err.println("---");
-        m_plugins = new ArrayList<SystemReportPlugin>(m_defaultServiceRegistry.findProviders(SystemReportPlugin.class));
+        m_plugins = new ArrayList<>(m_defaultServiceRegistry.findProviders(SystemReportPlugin.class));
         Collections.sort(m_plugins);
-        for (final SystemReportPlugin plugin : m_plugins) {
-            System.err.println(plugin.getName() + ":");
-            for (final Entry<String, org.springframework.core.io.Resource> entry : plugin.getEntries().entrySet()) {
-                System.err.println("  " + entry.getKey() + ": " + getResourceText(entry.getValue()));
-            }
-        }
```

Comment:
After my changes, ConfigurationReportPlugin started returning ByteArrayResources and this started clobbering the test output, which was throwing a very hard to read exception: https://app.circleci.com/pipelines/github/OpenNMS/opennms/16312/workflows/485271b0-9b9c-43f9-8dc1-c98f71a8b2d4/jobs/112774/parallel-runs/4/steps/4-104

I removed this logging so I could see the exception, but that seemed to fix the failure. Not sure why that happened.

I don't think this logging is necessary, so it might be better to just leave it out.